source/releasenotes/about.rst

```diff
@@ -17,79 +17,52 @@
-• Networker Backup Plugin for KVM Hypervisor
-• Custom Quota Tariffs
-• Secure VNC for KVM
+• CloudStack Object Storage Feature
+• VMware to KVM Migration
+• KVM Import
+• CloudStack DRS
+• OAuth2 Authentication
+• VNF Appliances Support
+• CloudStack DRS
+• CloudStack Snapshot Copy
+• Scheduled Instance Lifecycle Operations
+• Guest OS Management
+• Pure Flash Array and HPE-Primera Support
+• User-specified source NAT
+• Storage Browser
+• Safe CloudStack Shutdown
+• New CloudStack Dashboard
+• Domain migration
+• Flexible tags for hosts and storage pools
+• Support for Userdata in Autoscale Groups
+• KVM Host HA for StorPool storage
+• Dynamic secondary storage selection
+• Domain VPCs
+• Global ACL for VPCs
 
 The full list of new features can be found in the project release notes at
-https://docs.cloudstack.apache.org/en/4.18.0.0/releasenotes/changes.html
+https://docs.cloudstack.apache.org/en/4.19.0.0/releasenotes/changes.html
 
 .. _guestosids
 
-Possible Issue with Guest OS IDs
-================================
-
-It has been noticed during upgrade testing that some environment, where
-custom guest OSses where added and mapping for those OSses where added,
-problems may occur during upgrade. Part of the mitigation is to make sure
-OSses that are newly mapped but should have already been in the guest_os
-table are there. Make sure you apply those before you start the new 4.18
-management server.
-
-first check which of the guest_os entries you miss:
-
-.. parsed-literal::
-
-  SELECT * FROM cloud.guest_os WHERE display_name IN (´CentOS 8´, ´Debian GNU/Linux 10 (32-bit)´, ´Debian GNU/Linux 10 (64-bit)´, ´SUSE Linux Enterprise Server 15 (64-bit)´, ´Windows Server 2019 (64-bit)´)
-
-Then apply any of the following lines that you might need.
-
-.. parsed-literal::
+Possible Issue with volume snapshot revert with KVM
+===================================================
 
-  INSERT INTO cloud.guest_os (uuid, category_id, display_name, created, is_user_defined) VALUES (UUID(), '1', 'CentOS 8', now(), '0');
-  INSERT INTO cloud.guest_os (uuid, category_id, display_name, created, is_user_defined) VALUES (UUID(), '2', 'Debian GNU/Linux 10 (32-bit)', now(), '0');
-  INSERT INTO cloud.guest_os (uuid, category_id, display_name, created, is_user_defined) VALUES (UUID(), '2', 'Debian GNU/Linux 10 (64-bit)', now(), '0');
-  INSERT INTO cloud.guest_os (uuid, category_id, display_name, created, is_user_defined) VALUES (UUID(), '5', 'SUSE Linux Enterprise Server 15 (64-bit)', now(), '0');
-  INSERT INTO cloud.guest_os (uuid, category_id, display_name, created, is_user_defined) VALUES (UUID(), '6', 'Windows Server 2019 (64-bit)', now(), '0');
+Between versions 4.17.x, 4.18.0 and 4.18.1, KVM volume snapshot backups were
+not full snapshots and they rely on the snapshots on the primary storage.
+To prevent any loss of data, care must be taken during revert operation and
+it must be ensured that the source primary storage snapshot file is present
```

Comment:
Perhaps we can add more explicit checks on this part on how to find the file from database?

Reply:
@nvazquez done